fedimint-client/src/backup.rs

```diff
@@ -102,7 +102,11 @@ impl ClientBackup {
 
     /// Encrypt with a key and turn into [`EncryptedClientBackup`]
     pub fn encrypt_to(&self, key: &fedimint_aead::LessSafeKey) -> Result<EncryptedClientBackup> {
-        let encoded = Encodable::consensus_encode_to_vec(self);
+        let mut encoded = Encodable::consensus_encode_to_vec(self);
+
+        let alignment_size = Self::get_alignment_size(encoded.len());
+        let padding_size = alignment_size - encoded.len();
+        encoded.write_all(&vec![0u8; padding_size])?;
```

Comment:
I think this would break old clients, but that's ok as we don't support downgrades anyway.

Reply:
Yes, this would make the backup from the new client not work on an old one. Unsupported, unlikely and full recovery would still work.